start/src/main/java/org/apache/accumulo/start/Main.java

```diff
@@ -38,20 +37,17 @@ public class Main {
 
   private static final Logger log = LoggerFactory.getLogger(Main.class);
   private static ClassLoader classLoader;
-  private static Class<?> vfsClassLoader;
   private static Map<String,KeywordExecutable> servicesMap;
 
   public static void main(final String[] args) throws Exception {
     // Preload classes that cause a deadlock between the ServiceLoader and the DFSClient when
     // using the VFSClassLoader with jars in HDFS.
     ClassLoader loader = getClassLoader();
-    Class<?> confClass = null;
+    Class<?> confClass;
     try {
-      @SuppressWarnings("deprecation")
-      var deprecatedConfClass = org.apache.accumulo.start.classloader.AccumuloClassLoader
-          .getClassLoader().loadClass("org.apache.hadoop.conf.Configuration");
-      confClass = deprecatedConfClass;
-      Object conf = null;
+      confClass =
+          ClassLoader.getSystemClassLoader().loadClass("org.apache.hadoop.conf.Configuration");
+      Object conf;
```

Comment:
Pretty sure this stuff can now be removed. It was added for a very specific workaround that I don't think applies anymore: https://issues.apache.org/jira/browse/ACCUMULO-4341

Reply:
I can remove this but I was thinking that wouldn't this issue still be a problem if a user configured Accumulo to use the VFS classloader that is now external?

Reply:
Going back and looking at ACCUMULO-4341 and ACCUMULO-3923, it looks like this was added to get around an issue, that's been resolved, in VFS not loading resource files in jars correctly. I think this can be removed. If you test this with the new VFS classloader and we run into an issue, then we can put it back.